data/human/transport missions.txt

```diff
@@ -3209,3 +3209,97 @@ mission "Earth Retirement"
+	passengers 5 10
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination
+		distance 1 7
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "station"
+		not attributes "research"
+	to offer
+		random < 14
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 120000
+		dialog "As you descend through the upper atmosphere, the skydivers jump from your ship, spectacularly diving until they open their parachutes and land safely near the spaceport. You collect your payment of <payment>."
+
+
+
+mission "Spacediving professionals accident"
+	job
+	repeat
+	name "Spacediving at <planet>"
+	description "Head to <destination> with <bunks> skydivers, then drop them from the edge of space as you land. They are wearing thermal spacesuits, durable parachutes, and oxygen bottles to ensure their survival. Extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is 120000."
+	passengers 5 10
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination
+		distance 1 7
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "station"
+		not attributes "research"
+	to offer
+		"Spacediving professionals: done" > 7
+		random < 2
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		dialog "As you descend through the upper atmosphere, the skydivers jump from your ship, spectacularly diving until they try to open their parachutes. Some of the parachutes open only partially and part of the team suffers an uncontrolled hard landing. Medical teams respond quickly, but the skydivers are too preoccupied with their injured teammates to pay you the credits they owe you."
+
+
+
+mission "Spacediving professionals disaster"
+	job
+	repeat
+	name "Spacediving at <planet>"
+	description "Head to <destination> with <bunks> skydivers, then drop them from the edge of space as you land. They are wearing thermal spacesuits, durable parachutes, and oxygen bottles to ensure their survival. Extreme sports like skydiving typically carry more risks than regular passenger transports. Payment is 120000."
+	passengers 5 10
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination
+		distance 1 7
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		not attributes "station"
+		not attributes "research"
+	to offer
+		has "Spacediving professionals accident: done"
+		random < 1
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment -200000
```

Comment:
Missions can't complete if you don't have enough credits for a negative payment. I'd suggest checking how many credits the player has in the to offer and ensure that they have enough that they can pay this off, plus a little extra so that a player doesn't lose all their credits from taking this job, or so that the player doesn't have exactly what's needed but falls below that point before reaching the destination; this technically isn't foolproof, but it's at least a safety precaution.

Reply:
I added a check for 5000k (just a bit more safety margin than that you suggested).
Do you have a reference to the ideas and/or work for the fines?